src/lib/apiErrors.js

```diff
@@ -60,6 +60,18 @@ module.exports.NoUniversityIdMatch = class extends ApiError {
   }
 }
 
+module.exports.UserNotAuthenticated = class extends ApiError {
+  constructor (raw) {
+    super('User must be authenticated to access this resource', 102, 403, raw)
```

Comment:
Should a 401 be returned instead of a 403? http://stackoverflow.com/questions/3297048/403-forbidden-vs-401-unauthorized-http-responses